includes/util.py

```
@@ -0,0 +1,110 @@
+import sys
+import hashlib
+import math
+import subprocess
+import shlex
+
+BUF_SIZE = 1024*1024  # lets read stuff in 1MB chunks when hashing or copying
+
+def get_env(bench=False):
+    ''' Build dict of environment variables '''
+    env = dict()
+    if bench and cfgTuning['use_nosync']:
```

Ruff (0.14.13)

Undefined name `cfgTuning`

(F821)